management/src/test/resources/clusteredConfiguration.txt

```diff
@@ -0,0 +1,24 @@
+caches:
+    cache-1:
+        keyType: java.lang.String
+        valueType: java.lang.String
+        serviceConfigurations:
+        evictionAdvisor:
+        expiry: NoExpiry
+        resourcePools:
+            pools:
+                heap:
+                    size: 10 entries
+                    tierHeight: 10000
+                offheap:
+                    size: 1 MB
+                    tierHeight: 1000
+                clustered-dedicated:
+                    size: 1 MB (persistent)
+                    tierHeight: 10
+services:
+    - org.ehcache.clustered.client.config.ClusteringServiceConfiguration:
+        clusterUri: passthrough://server-1:9510/my-server-entity-1
+        readOperationTimeout: null
```

Comment:
I believe the decision on including `null` is going to be a case by case basis. But honestly, there should be no `null` in any configuration element as we have defaults in place normally - but for example, in the case here, I would indicate `MAX_LONG` when that is the value rather than the numerical value of that.